website_snippet_product_category/static/src/js/frontend.js

```diff
@@ -3,37 +3,38 @@
 odoo.define("website_snippet_product_category.s_product_category", function(require) {
     "use strict";
 
-    var core = require("web.core");
-    var sAnimation = require("website.content.snippets.animation");
+    const core = require("web.core");
+    const sAnimation = require("website.content.snippets.animation");
 
-    var _t = core._t;
+    const _t = core._t;
 
     sAnimation.registry.js_product_category = sAnimation.Class.extend({
         selector: ".js_product_category",
+        disabledInEditableMode: false,
```

Comment:
Did you check how the widget behaves in v12? It really doesn't let you edit anything.

Reply:
Yes, I test it, but if I don't use this key we can't see the preview

Reply:
Ok, I see :+1: